functional-test/pom.xml

```diff
@@ -516,6 +516,7 @@
               <classifier>mysql.classifier</classifier>
               <port>${mysql.port}</port>
               <socket>${mysql.socket}</socket>
+              <clearexistingdata>false</clearexistingdata>
```

Comment:
it is all lower case according to the plugin documentation

On Wed, Sep 23, 2015 at 1:50 PM, Alex Eng <notifications@github.com> wrote:

> In functional-test/pom.xml
> <https://github.com/zanata/zanata-server/pull/928#discussion_r40169186>:
>
> > @@ -516,6 +516,7 @@
> >                <classifier>mysql.classifier</classifier>
> >                <port>${mysql.port}</port>
> >                <socket>${mysql.socket}</socket>
> > +              <clearexistingdata>false</clearexistingdata>
>
> CamelCase?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/zanata/zanata-server/pull/928/files#r40169186>.
>



-- 
Patrick Huang
Senior Software Engineer
Engineering - Internationalisation
Red Hat, Asia-Pacific Pty Ltd
Level 1, 193 North Quay
Brisbane 4000
Office: +61 7 3514 8278
Fax: +61 7 3514 8199
IRC: pahuang
github: github.com/huangp
Website: www.redhat.com